packages/sui-studio/src/components/navigation/_style.scss

```diff
@@ -1,5 +1,5 @@
 .sui-StudioNav {
-  padding: 16px;
+  padding: 16px 16px 80px 16px;
```

Comment:
```suggestion
  padding: 16px 16px 80px;
```

Reply:
It's enough to set just one value for both sides

Reply:
I agree